packages/fury-adapter-oas3-parser/lib/parser/oas/parseComponentsObject.js

```diff
@@ -48,7 +48,16 @@ function parseComponentsObject(context, element) {
     const Member = R.constructN(2, namespace.elements.Member)(member.key);
     const parseResult = parseParameterObject(context, member.value);
     // Wrap non-annotation elements in member element
-    return R.map(R.unless(isAnnotation, Member), parseResult);
+    const result = R.map(R.unless(isAnnotation, Member), parseResult);
```

Comment:
It took me a while to read this, maybe because I was a bit out of context in the beginning. I'd at least rename `result` to something more descriptive, perhaps `parseResultMembers`.

Reply:
Good point, I've refactored significantly to try and address this. Please review https://github.com/apiaryio/api-elements.js/pull/122/commits/f80dd3f0afb8a699a6b5f1bbd0109cdf4d921cb3 and I'll follow up with any feedback separately.